source/community/release_notes/teak/aspects_updates.rst

```diff
@@ -1,15 +1,54 @@
+
+.. image:: /_images/release_notes/teak/in-context-highlevelengagementcharts.png
+
+Course delivery teams can easily drill in to view more detailed data about a single graded subsection, problem, or video by selecting the subsection or component of interest from the menu beneath the engagement charts.
+
+.. image:: /_images/release_notes/teak/in-context-subprobvid.png
+
+Component Engagement and Performance Analytics
+**********************************************
+
+In comparison to the analytics provided on the Course Outline page, the In-Context Analytics sidebar on the Unit page provides more granular insight into how learners are engaging with a single course component at a time. From the sidebar, users simply click on the component they want to view more information about to see more detailed data for that component.
+
+.. image:: /_images/release_notes/teak/in-context-unitsidebar.png
+
+Problem Analytics
+=================
+
+.. image:: /_images/release_notes/teak/in-context-problemcomponent.png
+
+For problem components, the In-Context Analytics sidebar shows the percentage of correct responses on the first problem attempt and on all problem attempts. The percentage of correct problem responses on the first attempt is a good indicator of how difficult the problem is for the learners that submitted a response for the problem. It is a good indicator of whether or not a learner immediately understands the question being asked and can identify the correct answer to the question. The percentage correct on all problem attempts is an indicator of how well learners were able to recover from earlier incorrect responses. A higher percentage correct out of all problem attempts indicates that the learner is able to figure out the right answer with additional effort or hints. If this percentage is still low, the problem may be too difficult or confusing for learners.
+
+The second table provides a breakdown of initial responses for each individual problem. The idea is to give course delivery teams a peak into learners’ thought processes when they approached the problem for the first time. This breakdown can give course authors quick insights into exactly how learners are getting a problem wrong, especially for difficult problems. This data can point to course delivery teams to where learners are getting mixed up.
+
+Video Analytics
+===============
+
+.. image:: /_images/release_notes/teak/in-context-videocomponent.png
+
+For video components, the In-Context Analytics sidebar displays the number of unique and repeat views for a single video by timestamp across the duration of the video. Timestamp ranges with a large number of repeat views should be reviewed as this might be an indicator that this particular section of video is unclear to learners.
+
+For more detailed information about each of the charts and tables provided out-of-the-box, please visit :ref:`openedx-aspects:reference/in_context_dashboards`.
```

Comment:
note to self this is likely going to break, fix at https://github.com/openedx/openedx-aspects/pull/327